e2e/ansible/playbooks/resiliency/test-node-eviction-out-of-disk/test-node-eviction-out-of-disk-vars.yml

```diff
@@ -0,0 +1,26 @@
+---
+test_name: test-node-eviction-out-of-disk
+
+utils_path: openebs/e2e/ansible/playbooks/utils
+
+patch_file: replica_patch.yaml
+
+create_sc: storage-class.yaml
+
+percona_link: https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/percona/demo-percona-mysql-pvc.yaml
+
+replica_count: 2 
```

Comment:
Can we arrive at this dynamically ? i.e., get the node count and decrease by 1 to obtain the replica count.

Reply:
Yeah, I will do that.

Reply:
Done @ksatchit Can you please have a look?